experiments/ssl4eo/run_ssl4eo_benchmark.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Runs the train script with a grid of hyperparameters."""
+import itertools
+import os
+import subprocess
+from multiprocessing import Process, Queue
+
+# list of GPU IDs that we want to use, one job will be started for every ID in the list
+GPUS = [1, 2, 3, 4]
```

Comment:
Can we rename gpu to device everywhere? For example, on macOS, I would use MPS, not GPU

Reply:
Can you do this in a big rename PR?